test/index.test.ts

```diff
@@ -0,0 +1,359 @@
+      external_link: 'https://ens.domains/name/sub1.wrappertest.eth',
+      attributes: [
+        {
+          display_type: 'date',
+          trait_type: 'Created Date',
+          value: 1623949711000,
+        },
+      ],
+    },
+  },
+  [mockNameHash.sub2]: {
+    domainResponse: {
+      domain: {
+        name: 'sub2.wrappertest9.eth',
+        id: mockNameHash.sub2,
+        labelName: 'sub2',
+        labelhash:
+          '0xb2fd3233fdc544d81e84c93822934ddd9b599f056b6a7f84f4de29378bf1cb15',
+        createdAt: '1625137830',
+        owner: { id: '0x97ba55f61345665cf08c4233b9d6e61051a43b18' },
+        parent: {
+          id: '0x0b00a980e17bfb715fca7267b401b08daa6e750f1bdac52b273e11c46c3e2b9f',
+        },
+        resolver: { texts: ['domains.ens.nft.image'] },
+        hasImageKey: true,
+      },
+    },
+    registrationResponse: null,
+    expect: {
+      name: 'sub2.wrappertest9.eth',
+      description: 'sub2.wrappertest9.eth',
+      image: 'https://i.imgur.com/JcZESMp.png',
+      image_url: 'https://i.imgur.com/JcZESMp.png',
+      external_link: 'https://ens.domains/name/sub2.wrappertest9.eth',
+      attributes: [
+        {
+          trait_type: 'Created Date',
+          display_type: 'date',
+          value: 1625137830000,
+        },
+      ],
+    },
+  },
+  [mockNameHash.unknown]: {
```

Comment:
Can you investigate what's the specified behaviour when the token does not exist? Should it return keys with empty value as we do now, should return just empty hash (`{}`) or 404?